zubhub_frontend/zubhub/package.json

```diff
@@ -4,10 +4,13 @@
   "private": true,
   "dependencies": {
     "@date-io/date-fns": "^1.3.13",
+    "@emotion/react": "^11.9.0",
```

Comment:
Why do we need this custom styling package? Can't we use something that is already available and being currently used in ZubHub?